pkg/config/sdk.go

```diff
@@ -28,22 +28,18 @@ type SdkInfo struct {
 	Hash       HashInfo                   `json:"hash,omitempty" yaml:"hash"`
 	Signature  SignatureInfo              `json:"signature,omitempty" yaml:"signature"`
 	Stream     StreamInfo                 `json:"stream,omitempty" yaml:"stream"`
+	StackLayer contracts.LayerType        `json:"layer,omitempty" yaml:"layer"`
```

Comment:
I think this should just be named `Layer` to be consistent with the Annotation property and the definition in the config json/yaml